macros.ddoc

```diff
@@ -2,9 +2,11 @@ _=Fundamental macros that apply to all generated formats
 
 ARGS = $0
 COMMA = ,
-COMMENT = 
+COMMENT =
```

Comment:
[It's part of our .editorconfig](https://github.com/D-Programming-Language/dlang.org/blob/master/.editorconfig), but I think we can override it for specific files if we need to.